channels/api.py

```diff
@@ -361,3 +366,50 @@ def more_comments(self, comment_fullname, parent_fullname, count, children=None)
         more_comments.submission = self.reddit.submission(submission_id)
         more_comments.comments()  # load the comments
         return more_comments
+
+    def add_moderator(self, moderator_name, channel_name):
+        """
+        Add a user to moderators for the channel
+
+        Args:
+            moderator_name(str): username of the user
+            channel_name(str): name of the channel
+
+        Returns:
+            praw.models.Redditor: the reddit representation of the user
+        """
+        try:
+            user = User.objects.get(username=moderator_name)
+        except User.DoesNotExist:
+            raise Exception("User {} does not exist".format(moderator_name))
```

Comment:
Can you change this to use `NotFound` similar to how it's used in https://github.com/mitodl/open-discussions/pull/140/files?